integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/EnvironmentEditTest.java

```diff
@@ -97,9 +99,9 @@ public void authenticationEditWhenSetManagedKeyAndNotDeleteExisted(MockedTestCon
                 .when(environmentTestClient.create())
                 .await(EnvironmentStatus.AVAILABLE)
 
-                .given(HttpMock.class)
-                .whenRequested(SpiEndpoints.RegisterPublicKey.class).post()
-                .thenReturn((s, model, uriParameters) -> "")
+//                .given(HttpMock.class)
```

Comment:
do you really want to comment out ?

Reply:
Yes, because these functionalities will be implemented in the next task (almost all verifications, response modifications)